src/diffusers/schedulers/scheduling_pndm.py

```diff
@@ -88,8 +96,10 @@ def __init__(
         beta_end: float = 0.02,
         beta_schedule: str = "linear",
         trained_betas: Optional[np.ndarray] = None,
-        tensor_format: str = "pt",
         skip_prk_steps: bool = False,
+        set_alpha_to_one: bool = True,
```

Comment:
Ah, I guess we want `False` for backward compatibility, even though it's not in line with DDIM?

Reply:
Yes we need `False` for backward compability indeed :-)